java.asciidoc

```diff
@@ -0,0 +1,298 @@
+= Pony for Java
```

Comment:
The contributing doc says one `=`:
https://github.com/ponylang/pony-for-x/blob/master/CONTRIBUTING.md

But I guess the AsciiDoc says two `==` for level one.
https://powerman.name/doc/asciidoc